src/Backend/Modules/Tags/Entity/Tag.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+namespace Backend\Modules\Tags\Entity;
+
+/*
+ * This file is part of Fork CMS.
+ *
+ * For the full copyright and license information, please view the license
+ * file that was distributed with this source code.
+ */
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * This is the Tag Entity
+ *
+ * @author Jeroen Desloovere <info@jeroendesloovere.be>
+ *
+ * @ORM\Entity
+ * @ORM\Table(name="tags")
```

Comment:
I'd remove this custom table name. In Symfony projects, it's mostly done with the automatic name.